api/v1/status_routes_test.go

```diff
@@ -103,8 +136,10 @@ func TestPatchStatus(t *testing.T) {
 	registry := metrics.NewRegistry()
 	builtinMetrics := metrics.RegisterBuiltinMetrics(registry)
 
-	for name, indata := range testdata {
+	for name, inData := range testdata {
```

Comment:
inData remember to me `inputData` but they are also `expectedData` so I would prefer a simple `td` as abbreviation of a generic `testdata`

Reply:
`td` is too short for the scope, but renamed the variable to the `testCase` since the `testData` is holding the table tests.